package.json

```
@@ -1,23 +1,49 @@
 {
   "name": "gatsby-plugin-pinterest",
-  "description": "Load the Pinterest JavaScript to let users Save images to their boards on hover",
-  "version": "1.1.0",
-  "author": "Robin Métral <robin@metral.ch>",
-  "bugs": {
-    "url": "https://github.com/robinmetral/gatsby-plugin-pinterest/issues"
-  },
-  "homepage": "https://github.com/robinmetral/gatsby-plugin-pinterest",
+  "version": "0.0.0-semantically-released",
+  "description": "Gatsby plugin to add Pinterest's embed script.",
```

The plugin isn't just for embeds, since a use case is to let Pinterest users save images to their boards

```suggestion
  "description": "Gatsby plugin to add the Pinterest script for embeds and save buttons",
```

---

I think using `add-on` is a better name, see https://developers.pinterest.com/docs/widgets/getting-started